### ISSUE TITLE
Deprecated functions

### ISSUE THREAD

**Issue by JuusoSaavalainen** [kassambara/survminer#638]
Function that produced the error:
 ```surv_summary(x)``` 

Warning msg:
```Warning message: 
`garher_()` was deprecated in tidyr 1.2.0
i Please use `gather()` instead.

**Comment by rmhirota** [comment]
`surv_median(x)` also produces the warning for deprecated functions:

```
Warning message:
`select_()` was deprecated in dplyr 0.7.0.
ℹ Please use `select()` instead.
```

**Comment by kassambara** [comment]
fixed in #579 and #665
